lib/tasks/install_secrets.rake

```diff
@@ -45,26 +45,40 @@ task :install_secrets, [] do
     end
   end
 
-  database_secrets = secrets.delete('database')
+  database_secrets = secrets.delete(:database)
+  write_yaml_file("config/database.yml", {
```

Comment:
seems like this sort of thing should go into a gem called "openstax-rails" or something...it's something all rails apps at openstax need to do for boilerplate eh?

Reply:
I would conceivable put the `write_yaml_file` in a gem, but not the specific lines here